test/k8sT/Nightly.go

```diff
@@ -74,25 +90,38 @@ var _ = Describe("NightlyEpsMeasurement", func() {
 		}
 		err := kubectl.WaitCleanAllTerminatingPods()
 		Expect(err).To(BeNil(), "Terminating containers are not deleted after timeout")
-	})
 
-	endpointCount := 20
-	manifestPath := "tmp.yaml"
-	vagrantManifestPath := path.Join(helpers.BasePath, manifestPath)
-	var lastServer int
+		kubectl.Delete(vagrantManifestPath)
+		kubectl.WaitCleanAllTerminatingPods()
+	})
 
-	Measure(fmt.Sprintf("%d endpoint creation", endpointCount), func(b Benchmarker) {
-		endpointsTimeout := endpointTimeout * time.Duration(endpointCount)
-		desiredState := string(models.EndpointStateReady)
-		var err error
+	deployEndpoints := func() {
 		_, lastServer, err = helpers.GenerateManifestForEndpoints(endpointCount, manifestPath)
-		Expect(err).Should(BeNil())
-
+		ExpectWithOffset(1, err).Should(BeNil(), "Manifest cannot be created correctly")
 		res := kubectl.Apply(vagrantManifestPath)
-		res.ExpectSuccess(res.GetDebugMessage())
+		res.ExpectSuccess("cannot apply eps manifest :%s", res.GetDebugMessage())
+	}
+
+	getServices := func() map[string]string {
```

Comment:
Even though this function isn't exported, add a comment indicating what this map represents.